packages/app/src/app/components/CodeEditor/Monaco/elements.ts

```diff
@@ -27,7 +26,7 @@ export const CodeContainer = styled.div`
       content: '';
       width: 50%;
       height: 50%;
-      background-color: ${theme.green()};
+      background-color: ${props => props.theme.green()};
```

Comment:
Same thing as button. but maybe this is not the case anymore. I will double check this!